docs/source/getting_started/overview.rst

```diff
@@ -3,15 +3,20 @@
 Overview
 --------
 
-JupyterLab is the next-generation user interface for Project Jupyter.
+JupyterLab is the next-generation web-based user interface for Project Jupyter.
 JupyterLab will eventually replace the classic Jupyter Notebook.
```

Comment:
IMHO, I wouldn't put this sentence so early in the document because this sentence puts pressure on users to transition during the beta.  I think we can mention this later in the document.

Reply:
Fair enough. I thought it was good to be up-front, but yeah, I can see the stress it could cause.

Reply:
Better with the last commit?